modules/cert-manager-proxy-support.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-proxy-support_{context}"]
+= Configuring proxy support for the {cert-manager-operator}
+
+// TODO: Is this title okay? Or should it be narrowed down to just injecting a custom CA?
```

Comment:
@thejasn @swghosh Is the main thing that they need to do is injecting the custom CA? If so, do you think I should make the title of this procedure more specific to injecting the custom CA?

Reply:
Yeah we can change the title. I would still want the following mentioned somewhere either as a note or a sub-section with a link to the OLM docs.
```
If a cluster wide egress proxy is configured on the OpenShift cluster, OLM automatically update all the operators' deployments with HTTP_PROXY, HTTPS_PROXY, NO_PROXY environment variables.
Those variables are then propagated down to the cert-manager (operand) controllers by the cert manager operator.
```

Reply:
@thejasn thanks, I updated the title to "Injecting a custom CA certificate for the cert-manager Operator for Red Hat OpenShift". Let me know if that works.

As for your note, I do have most of that information at the top of the section, let me know if that covers what you were looking for: https://56754--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-proxy.html